packages/react/src/schematics/application/files/app/src/app/__fileName__.spec.tsx__tmpl__

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { render, cleanup } from 'react-testing-library';
+import { render, cleanup } from '@testing-library/react';
```

Comment:
Can we write a migration to fix this for people? + add the new package

Example:
https://github.com/nrwl/nx/blob/master/packages/schematics/migrations/update-8-0-0/update-8-0-0.ts#L168